akka-actor/src/main/scala/akka/actor/ActorSystem.scala

```diff
@@ -57,6 +119,19 @@ object ActorSystem {
   def create(name: String): ActorSystem = apply(name)
 
   /**
+   * Java API: Creates a new actor system with the specified name and settings
+   * The core actor system settings are defined in [[ActorSystemBootstrapSettings]]
+   */
+  def create(name: String, settings: ActorSystemSettings): ActorSystem = apply(name, settings)
+
+  /**
+   * Java API: Shortcut for creating an actor system with custom bootstrap settings.
+   * Same behaviour as calling `ActorSystem.create(name, ActorSystemSettings.create(bootstrapSettings))`
+   */
+  def create(name: String, bootstrapSettings: ActorSystemBootstrapSettings): ActorSystem =
```

Comment:
After using it a little bit having to construct the `ActorSystemSettings` I think it feels a bit boilerplatey, would want to have a `create/apply(String, ActorSystemSetting, ActorSystemSetting*)`, but that would conflict with this one.

```
ActorSystem("whatever", RemotingSettings("localhost", 666))
ActorSystem("duh", ActorSystemBootstrapSettings(config))
```

Feels more reasonable than
```
ActorSystem("whatever", ActorSystemSettings(RemotingSettings("localhost", 666)))
ActorSystem("duh", ActorSystemSettings(ActorSystemBootstrapSettings(config)))
```

Maybe also change `ActorSystemBootstrapSettings` -> `BootstrapSettings`, it will anyways be scoped by package `akka.actor`

Opinions?